lighthouse-viewer/gulpfile.js

```diff
@@ -83,7 +83,6 @@ gulp.task('browserify', () => {
           removeComments: true
         })
         .transform('brfs')
-        .ignore('../lighthouse-core/lib/log.js')
```

Comment:
think viewer still wants to omit logger from the brfs output.

`.ignore('../lighthouse-logger/index.js')`?